docs/UserGuide.md

```diff
@@ -9,331 +9,424 @@ Interface** (CLI) while still providing an easy way to visualize all events thro
 
-Format: `undo`
+Displays all events currently saved in the scheduler.
 
-Example: `undo`<br><br>
-
-[[Back to top](#index)]
+Format: `list`  
+Example: `list`
 
+[[Back to top](#table-of-contents)]
 
-<h3 id="help">View Help</h3>
+<h3 id="help">View help</h3>
 
 Provide a link to access this website.
 
-Format: `help`
-
-Example: `help`<br><br>
+Format: `help`  
+Example: `help`
 
-[[Back to top](#index)]
+[[Back to top](#table-of-contents)]
 
-
-<h3 id="exit">Exit Application</h3>
+<h3 id="exit">Exit application</h3>
 
 Exits the application.
 
-Format: `exit`
-
-Example: `exit`<br><br>
+Format: `exit`  
+Example: `exit`
 
-[[Back to top](#index)]
+[[Back to top](#table-of-contents)]
 
+-------------------------------
 
-<h2 id="gui">GUI</h2>
+<h2 id="gui">Graphical User Interface</h2>
 
 <h3 id="command-box">Command Box</h3>
 ![command-box.png](images/command-box.png)
-Allows you to type the command in the command box and press Enter to execute the command. 
-E.g., typing help and pressing Enter will open the help window
-
-[[Back to top](#index)]
+This feature enables you to type the command in the command box and execute it by pressing Enter  
+E.g., typing `help` and pressing Enter will open the help window.
 
+[[Back to top](#table-of-contents)]
 
 <h3 id="response-box">Response Box</h3>
 ![response-box.png](images/response-box.png)
-Displays to you the feedback of the command you typed into the command box.
 
-[[Back to top](#index)]
+This feature displays the feedback of the command you entered into the command box.
 
+[[Back to top](#table-of-contents)]
 
-<h3 id="event-list">Event List</h3>
+<h3 id="events-panel">Events Panel</h3>
 ![event-list.png](images/event-list.png)
-Displays to you all the events that you have.
-Use commands such as `list` and `find` to show you the events you have in your list.
+
+This feature displays all the events that you have.  
+You can use commands like `list` and `find` to view the events in your scheduler.
 
 <div markdown="span" class="alert alert-info">
 :information_source: **Note:**  
 Events that have past is marked with a completed label and check mark.   
 Events are displayed in chronological order.  
 </div>
 
-[[Back to top](#index)]
-
+[[Back to top](#table-of-contents)]
 
-<h3 id="upcoming">Upcoming Events</h3>
+<h3 id="upcoming-events-panel">Upcoming Events Panel</h3>
 ![upcoming-events.png](images/upcoming-events.png)
-Displays ongoing (in case you have not start on it yet) and upcoming events you have. 
 
-<div markdown="span" class="alert alert-info">
+This feature displays both ongoing (if you have not started on them yet) and upcoming events that you have.
+
+<div markdown="span" class="alert alert-success">
 :bulb: **Tip:**
-While the default is one upcoming event,
-you can use the `next` command to change the number of upcoming events shown.
+While the default is one upcoming event, you can use the `next` command to change the number of upcoming events shown.
```

Comment:
Mentioned previously, lines broken up to increase readability (esp. when I'm IDEA splits the screen into half); does not affect actual UI.